integration_tests/tests/rpc/conftest.py

```diff
@@ -5,3 +5,10 @@
 @pytest.fixture(scope="module")
 def client(network: ConfluxTestFramework) -> RpcClient:
     return network.rpc
+
+# espace rpc method caller
+@pytest.fixture(scope="module")
+def call_rpc(ew3):
```

Comment:
Not appropriate to define util as fixture. 
1. You can use `client` fixture to directly call rpc methods
2. Or if you do want to use this method, define a util function in integration_tests/utils/.. would be better